deel-1-flexbox-positioneren/styles/styles.css

```diff
@@ -1 +1,78 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
```

Comment:
Ik mis hier de css-reset om de margin en padding op 0 in te stellen en de box-sizing op border-box